setup.py

```diff
@@ -32,8 +32,17 @@
         if not line.startswith("#"):
             fuseimg_requirements.append(line.strip())
 
+# list of requirements for fuse_examples
+fuse_exampels_requirements = []
+with open(os.path.join(HERE, "fuse_examples/requirements.txt"), "r") as fh:
+    for line in fh:
+        if not line.startswith("#"):
+            fuseimg_requirements.append(line.strip())
```

Comment:
should be `fuse_examples_requirements.append` right?

Reply:
Nice catch! Thanks.